backend/tests/test_benchmarks.py

```
@@ -37,5 +37,55 @@ def test_get_memory_usage(self, mock_process):
 
+    def test_get_memory_usage_returns_correct_value(self):
+        """Test that memory usage returns the correct value in MB."""
+        import backend.benchmarks
+        if hasattr(backend.benchmarks, 'get_memory_usage'):
+            with patch('backend.benchmarks.psutil.Process') as mock_process:
+                mock_memory_info = MagicMock()
+                mock_memory_info.rss = 1024 * 1024 * 256  # 256 MB
+                mock_process.return_value.memory_info.return_value = mock_memory_info
+
+                memory = get_memory_usage()
+                self.assertAlmostEqual(memory, 256.0, places=1)
```

Ruff (0.15.2)

[warning] Use a regular `assert` instead of unittest-style `assertAlmostEqual`

(PT009)